hazelcast/src/main/java/com/hazelcast/config/DataConnectionConfig.java

```diff
@@ -181,6 +183,25 @@ public DataConnectionConfig setProperties(Properties properties) {
         return this;
     }
 
+    /**
+     * Checks if this configuration object is in coherent state - has all required properties set.
+     */
+    public void checkValidity() {
```

Comment:
why not simply `public void validate() {` ? ;)

Reply:
And, do we need to have this in the public API? Maybe we can also put `@PrivateApi` annotation here